### PR TITLE
New W3C Inc. copyright

### PR DESCRIPTION
**Pull request by plehegar**
This is the new copyright of W3C, starting January 1st 2023.

Do not merge before January 2023.

### CODE REVIEW

**Comment by plehegar** [comment]
https://github.com/w3c/specberus/issues/1681

File: boilerplate/w3c/copyright.include [hunk 1]
```diff
@@ -1 +1 @@
...
+<a href="https://www.w3.org/Consortium/Legal/ipr-notice#Copyright">Copyright</a> © [YEAR] <a href="https://www.w3.org/">World Wide Web Consortium, Inc.</a>. <abbr title="World Wide Web Consortium">W3C</abbr>®</sup> <a href="https://www.w3.org/Consortium/Legal/ipr-notice#Legal_Disclaimer">liability</a>, <a href="https://www.w3.org/Consortium/Legal/ipr-notice#W3C_Trademarks">trademark</a> and <a rel="license" href="https://www.w3.org/Consortium/Legal/2015/copyright-software-and-document">permissive document license</a> rules apply.
```
Comment (annevk) [inline boilerplate/w3c/copyright.include:1]: That lone `</sup>` looks bogus.

Also, are you sure you want that comma there? It's not in the title of this PR...

Comment (annevk) [reply boilerplate/w3c/copyright.include]: I meant the comma before Inc., to be clear. (The superscript also seems somewhat redundant to me given the character used, but I guess that is how it is done already.)

Comment (plehegar) [reply boilerplate/w3c/copyright.include]: so far, the information I have tells me it's "World Wide Web Consortium, Inc" but I'm double checking to be sure.

On the <sup>, it came from the previous markup. Looking around, it does seem indeed that the (R) symbol isn't used with superscript, unlike (tm) or (sm).

Comment (plehegar) [reply boilerplate/w3c/copyright.include]: So, it's really "[World Wide Web Consortium, Inc.](https://opencorporates.com/companies/us_de/7614104)".

Comment (plehegar) [reply boilerplate/w3c/copyright.include]: and for ®, the float above the baseline varies depending on the font used. So I guess we should stick with <sup> for now.

File: boilerplate/w3c/copyright.include [hunk 2]
```diff
@@ -1 +1 @@
...
+<a href="https://www.w3.org/Consortium/Legal/ipr-notice#Copyright">Copyright</a> © [YEAR] <a href="https://www.w3.org/">World Wide Web Consortium, Inc.</a>. <abbr title="World Wide Web Consortium">W3C</abbr><sup>®</sup> <a href="https://www.w3.org/Consortium/Legal/ipr-notice#Legal_Disclaimer">liability</a>, <a href="https://www.w3.org/Consortium/Legal/ipr-notice#W3C_Trademarks">trademark</a> and <a rel="license" href="https://www.w3.org/Consortium/Legal/2015/copyright-software-and-document">permissive document license</a> rules apply.
```
Comment (deniak) [inline boilerplate/w3c/copyright.include:1]: Following internal discussion, the ", Inc." should be dropped from the copyright. Another email will follow on spec-prod to explain this decision.
```suggestion
<a href="https://www.w3.org/Consortium/Legal/ipr-notice#Copyright">Copyright</a> © [YEAR] <a href="https://www.w3.org/">World Wide Web Consortium</a>. <abbr title="World Wide Web Consortium">W3C</abbr><sup>®</sup> <a href="https://www.w3.org/Consortium/Legal/ipr-notice#Legal_Disclaimer">liability</a>, <a href="https://www.w3.org/Consortium/Legal/ipr-notice#W3C_Trademarks">trademark</a> and <a rel="license" href="https://www.w3.org/Consortium/Legal/2015/copyright-software-and-document">permissive document license</a> rules apply.
```

**Review by deniak — CHANGES_REQUESTED** [review]

**Comment by tabatkins** [comment]
I've been waiting for confirmation from one of the editors if the PR that is finished, given the discussion/fixes that went in a bit ago.

**Comment by annevk** [comment]
Right, there's still also an outstanding comment by @deniak.

**Comment by svgeesus** [comment]
About dropping the "Inc."? The checker already fails specs if "W3C, Inc." is present.

**Comment by annevk** [comment]
That's great, but the PR hasn't been updated to reflect that.

File: boilerplate/w3c/copyright.include (hunk 2 above)
Comment (deniak) [reply boilerplate/w3c/copyright.include:1]: @plehegar can you please review/integrate this suggestion?

**Comment by plehegar** [comment]
As far as I can tell, this PR is now ready to be merged.

**Review by annevk — APPROVED** [review]
I'm a bit surprised there's no Oxford comma, but that's an existing issue.

**Pull request merged**